CRM/Core/BAO/OptionValue.php

```diff
@@ -192,7 +193,7 @@ public static function add(&$params, $ids = []) {
       $p = [1 => [$params['option_group_id'], 'Integer']];
 
       // Limit update by domain of option
-      $domain = $optionValue->domain_id ?? NULL;
+      $domain = CRM_Utils_System::isNull($optionValue->domain_id) ? NULL : $optionValue->domain_id;
```

Comment:
I was confused at first thinking this would give an error in the typical situation, where `$optionValue->domain_id` never gets set and so attempting to reference a non-existent thing, but it's declared as a property in the DAO so works ok. So the previous `??` wasn't really necessary.

Reply:
Ditto :)